controller/vars.tf

```diff
@@ -0,0 +1,49 @@
+variable "region" {}
+
+variable "vpc" {}
+
+variable "subnet" {}
+
+variable "keypair" {}
+
+variable "ec2role" {}
+
+#
+# Defaults
+#
+
+# This is the default root volume size as suggested by Aviatrix
+variable "root_volume_size" {
+  default = 16
+}
+
+variable "root_volume_type" {
+  default = "standard"
+}
+
+variable "incoming_ssl_cidr" {
+  default = "0.0.0.0/0"
```

Comment:
Default is to open up the world, but if this is not desired, it allows a client using the module to over-ride as necessary.